### PR TITLE
Release 4.2.1

### PR DESCRIPTION
**Pull request by irojkov-ph**
Complete Milestone v4.2.1 with updated documents.

### CODE REVIEW

File: doc/PsPM_Manual.lyx [hunk 1]
```diff
@@ -151,7 +151,7 @@ literal "false"
 \size larger
 Dominik R Bach, Giuseppe Castegnetti, Laure Ciernik, Samuel Gerster, Saurabh
  Khemka, Christoph Korn, Tobias Moser, Philipp C Paulus, Matthias Staib,
```
Comment (eozd) [inline doc/PsPM_Manual.lyx]: I think that is alphabetical. Let us not discontinue the convention.

File: src/pspm_msg.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
 $___________________________________________________________________________
 Welcome to PsPM - PsychoPhysiological Modelling
 (incorporating SCRalyze)
-Version 4.2.0 (17.06.2019)
+Version 4.2.1 (17.06.2019)
```
Comment (eozd) [inline src/pspm_msg.txt]: Please also update the date. I think we can release today, so let us use today's date.

File: src/pspm_quit.m [hunk 3]
```diff
@@ -1,6 +1,6 @@
 % pspm_quit clears settings, removes paths & closes figures
 %__________________________________________________________________________
-% PsPM 4.2.0
+% PsPM 4.2.1
```
Comment (eozd) [inline src/pspm_quit.m]: In `pspm_quit.m`, version should be update at line 36.

**Comment by irojkov-ph** [comment]
Yep its done

**Pull request merged**